cumulus/pallets/executor-registry/src/lib.rs

```diff
@@ -419,6 +419,61 @@ mod pallet {
         }
     }
 
+    type GenesisExecutorInfo<T> = (
+        <T as frame_system::Config>::AccountId,
+        BalanceOf<T>,
+        <T as frame_system::Config>::AccountId,
+        ExecutorId,
+    );
+
+    #[pallet::genesis_config]
+    pub struct GenesisConfig<T: Config> {
+        pub executors: Vec<GenesisExecutorInfo<T>>,
+    }
+
+    #[cfg(feature = "std")]
+    impl<T: Config> Default for GenesisConfig<T> {
+        fn default() -> Self {
+            Self {
+                executors: Vec::new(),
+            }
+        }
+    }
+
+    #[pallet::genesis_build]
+    impl<T: Config> GenesisBuild<T> for GenesisConfig<T> {
+        fn build(&self) {
+            for (executor, initial_stake, reward_address, executor_id) in self.executors.clone() {
+                assert!(
+                    initial_stake >= T::MinExecutorStake::get()
+                        && initial_stake <= T::MaxExecutorStake::get(),
+                    "Initial stake can not be too small or too large"
+                );
```

Comment:
Interestingly you have min/max number of executors, but it is not checked here. Is there a reason for it?